mobile/src/bundles/talent/store/slice.ts

```diff
@@ -2,94 +2,65 @@ import { createSlice, isAnyOf } from '@reduxjs/toolkit';
 
 import { DataStatus } from '~/bundles/common/enums/enums';
 import { type ValueOf } from '~/bundles/common/types/types';
-import {
-    type BadgeStepDto,
-    type ProfileStepDto,
-    type SkillsStepDto,
-} from '~/bundles/talent/types/types';
+import { type UserDetailsGeneralResponseDto } from '~/bundles/talent/types/types';
 
 import {
-    completeBadgesStep,
-    completeProfileStep,
-    completeSkillsStep,
+    createTalentDetails,
+    getTalentDetails,
+    setCompletedStep,
+    updateOnboardingData,
 } from './actions';
 
 type State = {
     dataStatus: ValueOf<typeof DataStatus>;
-    profileStepData: ProfileStepDto | null;
-    badgesStepData: BadgeStepDto | null;
-    skillsStepData: SkillsStepDto | null;
+    onboardingData: UserDetailsGeneralResponseDto | null;
+    completedStep: string | null;
```

Comment:
completedStep: ValueOf<typeof TalentOnboardingScreenName> | null;

Reply:
it is already included in onboardingData